Source/WebCore/css/CSSSelector.cpp

```diff
@@ -428,21 +428,21 @@ static void outputNthChildAnPlusB(const CSSSelector& selector, StringBuilder& bu
 
 String CSSSelector::selectorText(StringView separator, StringView rightSide) const
 {
-    StringBuilder builder;
-
-    auto serializeIdentifierOrStar = [&] (const AtomString& identifier) {
+    auto serializeIdentifierOrStar = [&](const AtomString& identifier, StringBuilder& builder) {
```

Comment:
Maybe we can write `[]` instead of `[&]` now since this doesn’t need to capture anything?

Reply:
Do I not still need it to capture `serializeIdentifier` ?

Reply:
Actually I don't.